src/items/powerup_manager.cpp

```diff
@@ -232,12 +229,19 @@ void PowerupManager::WeightsData::readData(int num_karts, const XMLNode *node)
 
         // Keep a reference for shorter access to the list
         std::vector<int> &l = m_weights_for_section.back();
-        for(unsigned int i=0; i<l_string.size(); i++)
+        for(unsigned int j=0; j < l_string.size(); ++j)
```

Comment:
Same remark about ++j vs j++

Switching to j instead of i is definitely good for code clarity.